lms/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "lms-signature"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+name = "lms"
+path = "lms/lib.rs"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0.75"
```

Comment:
`anyhow` is typically used for application-level error handling as opposed to libraries

Reply:
It also looks like it only has one usage here: https://github.com/RustCrypto/signatures/pull/801/files#diff-4a60ec7cc2aeac6ce05a26ae2a5b3d69d23c199b6d4f9a93f2a2bef5ef90ba88R106